packages/telegram-cloud-storage-stamper/src/util/telegram-environment.ts

```diff
@@ -0,0 +1,17 @@
+import JSDOMEnvironment from "jest-environment-jsdom";
+
+class TelegramEnvironment extends JSDOMEnvironment {
+  override async setup() {
+    await super.setup();
+
+    this.global.window.Telegram = {
+      WebApp: {},
+    };
```

Comment:
I think you might need "CloudStorage" in there as well now that the feature detection code changed.

Reply:
changed